src/BookNavigator/visual-adjustments/visual-adjustments-provider.js

```diff
@@ -60,7 +60,7 @@ export default class {
   }
 
   onZoomOut() {
-    this.bookreader.zoom();
+    this.bookreader.zoom(-1);
```

Comment:
thanks for being explicit with zoom direction

Reply:
I think this is the only thing necessary to fix the bug; put this in its own commit. For the refactor you want to w.r.t zoom argument types, have that be a separate commit. Makes the history clearer :)

Reply:
@cdrini yes you are correct, on both counts.